src/styles/mixins.scss

```diff
@@ -6,10 +6,6 @@
   background-color: transparent;
   cursor: pointer;
   font-size: 16px;
-
-  &:focus {
-    outline: 0;
```

Comment:
Instead of removing the :focus completely, add a font-weight: bold below the outline. I think it still helps with accessibility while not making the layout terrible.

Reply:
Not entirely sure if I agree. If you add bold it only highlights text. I can' see the menu button being focused or any other element that doesn't have text. I now use the text color CSS var for focus, meaning you see a black outline or white depending on light/dark mode. Way less intrusive. However, still not perfectly visible. Could also look at doing tab detection so tabbing users will see the original blue focus, but non-tabbing users wont; https://medium.com/hackernoon/removing-that-ugly-focus-ring-and-keeping-it-too-6c8727fefcd2